src/parser/generated_parser.ts

```diff
@@ -645,7 +647,7 @@ export class GeneratedParser extends Parser {
         let a, b, mark;
         mark = this.mark();
         if ((a = this.name()) && ((b = this._tmp_35()), 1)) {
-            return new astnodes.alias(a.id, b.id);
+            return new astnodes.alias(a.id, b ? b.id : null);
```

Comment:
you'd also revoked this `?` but `b` can still be `null` right

Reply:
Yeah this is better. I changed it in #12  to `b?.id` just because.
But then I had to keep an action cleaner to convert `? .` to `?.`. See https://github.com/skulpt/skulpt_parser/pull/12/files#r654725909